libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -579,6 +574,11 @@ void PbftManager::broadcastVotes() {
 
       LOG(log_dg_) << "Broadcast own votes for period " << period << ", round " << round;
     }
+
+    // Broadcast own pillar vote
+    if (const auto &own_pillar_vote = db_->getOwnPillarBlockVote(); own_pillar_vote) {
+      net->gossipPillarBlockVote(own_pillar_vote, rebroadcast);
```

Comment:
manager does not store it in memory either... own pillar block vote is accessed only in pillar manager ctor or in pbft manager when the node is stalled.... so during normal node operation it is not needed in memory at all and for the edge case of node being stalled it might not be worth it ?